### ISSUE TITLE
[A11y]

### ISSUE THREAD

**Issue by hakosch** [IBMa/equal-access#146]
## Environment

> Operating system:

> Browser:

> Automated testing tool and ruleset:

> Assistive technology used to verify:

## Detailed description

> What WCAG 2.1 checkpoint does the issue violate?

## Steps to reproduce the issue

1. Step one
2. Step two
3. Step three
4. etc.

## Additional information

- Screenshots or code
- Notes
